### PR TITLE
feat: integration tests with docker-compose

### PR DESCRIPTION
**Pull request by aludwiko**
References #xxxx

### CODE REVIEW

File: sdk/java-sdk-spring/src/it/java/com/example/wiring/pubsub/DockerIntegrationTest.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+    Config config = ConfigFactory.parseMap(confMap).withFallback(ConfigFactory.load());
+
+    kalixSpringApplication = new KalixSpringApplication(applicationContext, config);
+  }
+
+  @BeforeAll
+  public void beforeAll() {
+    dockerComposeUtils.start();
+    kalixSpringApplication.start();
+    //http://localhost:59644/counter/helloRestart/increaseaaaa/15 [DefaultWebClient]
```
Comment (octonato) [inline sdk/java-sdk-spring/src/it/java/com/example/wiring/pubsub/DockerIntegrationTest.java:81]: increaseaaaa?

**Review by octonato — COMMENTED** [review]
Looking good. 

Anything missing to make it non-draft?

File: sdk/java-sdk-spring/src/it/java/com/example/wiring/pubsub/DockerIntegrationTest.java (hunk 1 above)
Comment (aludwiko) [reply sdk/java-sdk-spring/src/it/java/com/example/wiring/pubsub/DockerIntegrationTest.java:81]: hehe, I was testing sth

**Comment by aludwiko** [comment]
> Anything missing to make it non-draft? 

yeah it's ready, testing s2s eventing is done in samples, so with these additional tests we have more coverage

**Review by octonato — APPROVED** [review]

**Pull request merged**